scalding-db/src/main/scala/com/twitter/scalding/db/MySqlJdbcWriter.scala

```diff
@@ -0,0 +1,152 @@
+/*
+Copyright 2015 Twitter, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package com.twitter.scalding.db
+
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{ FileSystem, Path }
+import org.apache.hadoop.mapred.JobConf
+import org.json4s._
+import org.json4s.native.JsonMethods._
+import org.slf4j.LoggerFactory
+
+import com.twitter.scalding.db.driver.DriverClass
+
+import java.sql._
+import scala.annotation.tailrec
+import scala.io.{ Codec, Source }
+import scala.util.Try
+
+class MySqlJdbcWriter[T](
```

Comment:
can we get some comments here. What is json2CaseClass doing? The role of JSON is unclear to me.